nomad/local/nomad_cli_tools.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Before reviewing `nomad_cli_tools.sh` i suggest reviewing `nomad/local/run_integration_tests.sh` instead, which shows you _how_ each of these gets used

Reply:
I suspect that we'll need to refactor some of this in the future given that it makes a number of assumptions that may not hold true forever. That said its definitely good enough for now